src/openms/include/OpenMS/ANALYSIS/ID/ConsensusIDAlgorithm.h

```diff
@@ -28,71 +28,83 @@
 // ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 // --------------------------------------------------------------------------
-// $Maintainer: Sven Nahnsen $
-// $Authors: Andreas Bertsch, Marc Sturm, Sven Nahnsen $
+// $Maintainer: Hendrik Weisser $
+// $Authors: Andreas Bertsch, Marc Sturm, Sven Nahnsen, Hendrik Weisser $
 // --------------------------------------------------------------------------
 
-#ifndef OPENMS_ANALYSIS_ID_CONSENSUSID_H
-#define OPENMS_ANALYSIS_ID_CONSENSUSID_H
+#ifndef OPENMS_ANALYSIS_ID_CONSENSUSIDALGORITHM_H
+#define OPENMS_ANALYSIS_ID_CONSENSUSIDALGORITHM_H
 
 #include <OpenMS/DATASTRUCTURES/DefaultParamHandler.h>
 #include <OpenMS/METADATA/PeptideIdentification.h>
 
+#include <map>
 #include <vector>
 
 namespace OpenMS
 {
   /**
-    @brief Calculates a consensus ID from several ID runs
+    @brief Base class for all ConsensusID algorithms (that calculate a consensus from multiple ID runs)
 
-    This class combines several ID runs using one of several, available algorithms.
-
-        @htmlinclude OpenMS_ConsensusID.parameters
-
-        @ingroup Analysis_ID
+    @htmlinclude OpenMS_ConsensusIDAlgorithm.parameters
```

Comment:
can you maybe write that this is an abstract base class and which functions are to be implemented by its child classes (e.g. `apply_`)